OneCommunityGlobal/HighestGoodNetworkApp
# Vishala log materials bm dashboard

**# Description**
The UI design for Log Materials component is implemented in this PR.
Implements # Phase 2 , WBS : 3.3.3

**## Main changes explained:**
- Created 3 jsx files and 1 css file to build the UI of Log materials page

**## How to test:**
1. check into current branch
2. do `npm run start:local` to run this PR locally
3. log as admin user
4. go to (http://localhost:3000/bmdashboard/logMaterials)
5. verify if the UI matches with the wireframe design attached and check for responsiveness

**## Note:**
Using the same wireframe design of Tool/Equipment for Log Materials. Hence the terms used in the current branch will be related to Log Materials

**## Screenshots or videos of changes:**
Wireframe Design:
<img width="527" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/65274029/f9a86fd3-6622-418b-bc00-e628157d37b8">

------------------------------------------------------------------------------------------------------------

**UI Changes:**
<img width="960" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/65274029/6212700f-7132-4f11-8ebd-1aff4636735c">

<img width="960" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/65274029/eb63c3f0-c833-4b41-8686-b04cda0bf66f">
